scripts/ball.gd

```diff
@@ -3,7 +3,9 @@ extends RigidBody2D
 export var points_on_hit   = 250
 
 export var speedup   = 5
-export var max_speed = 1500
+export var max_speed = 2000
+
+const PUP_TELE_ACC = 1000
```

Comment:
Should we default all constants, be it `var` or `const`, to lower case or upper case?

I suggest you make this an `export var`, so its settable inside the scene editor.

Finally, I suggest a more descriptive name, for example `powerup_telecontrol_acceleration`. Seems more readable to me.